resource-managers/cook/src/main/scala/org/apache/spark/scheduler/cluster/cook/CookSchedulerContext.scala

```diff
@@ -46,6 +46,8 @@ case class CookSchedulerContext(
 
   val SPARK_EXECUTOR_COOK_CONTAINER = "spark.executor.cook.container"
 
+  val SPARK_EXECUTOR_COOK_POOL = "spark.executor.cook.pool"
```

Comment:
`cook.pool` is a driver property, we could name it `spark.cook.pool`

Reply:
It's a job property like cores/mem/container. Not to be confused with the cook scheduler URL

Reply:
Haha :-), that is different things. The Spark driver constructs the Cook job client and send the request to Cook. The pool feature is property that only a driver can see. So that is why it is a driver property. By Spark's convention, it should be `spark.cook.pool` similar to `spark.cook.priority` property

Reply:
Got it - fixed.